theme/_variables.scss

```diff
@@ -1,5 +1,6 @@
 // Theme Variable overrides
 
-// Add variables here
+//Breadcrumb list background colour (Boostrap override)
+$breadcrumb-bg: #FFF;
```

Comment:
@nschonni 

* I'll change the colour code into lower-case soon.
* ``$wb-bc-background-color`` is set on ``<nav id="wb-bc">``. It controls the breadcrumb's background colour outside the content area. It was originally ``#f9f9f9``, but that's already the default background colour of the WET theme's ``<body>`` element, so IMO there's no value in re-declaring it.
* ``$breadcrumb-bg`` is a Bootstrap variable that's used on ``<ol class="breadcrumb">``. It controls the breadcrumb's background colour inside the content area. Most themes don't override Bootstrap's default colour for this (the WET and base themes are the exception).